noxfile.py

```diff
@@ -0,0 +1,19 @@
+import nox_poetry
+
+
+@nox_poetry.session
+def lint(session):
+    session.install("flake8")
+    session.run("flake8", "coffee_roaster_market", "manage.py")
+
+
+@nox_poetry.session
+def black(session):
+    session.install("black")
+    session.run("black", "--check", "coffee_roaster_market")
+
+
+@nox_poetry.session
+def tests(session):
+    session.run("poetry", "install", external=True)
```

Comment:
@kuchichan 
If we want to call `poetry install` explicite in nox file, we probably don't need nox-poetry. However if we would use nox-poetry package, we could add "flake8" and "black" packages with desired versions as dev-dependencies in toml. (is it really needed?) Nox poetry is only needed if we want to specify desired packages in each step (i.e. `session.install("Django")`) and it would use versions defined in toml file and poetry.lock. According to nox-poetry documentation such solution is better and more readable. What I don't like in this, is duplication of desired packages both in toml file and noxfile.
What do you think? Should we use nox-poetry and define packages for test step in noxfile or just call `poetry isntall`?
https://nox-poetry.readthedocs.io/en/latest/

Reply:
I saw that, I have even ran that. Please note, that we probably do not need very elaborate package management (i.e different python versions and so on) because we will be running django (wsgi) server with some one particular version). In the other hand, we will not have such a headache with duplicated black / flake8 definitions - we will just set it once probably. And for that reason I would stick to the nox_poetry version. And the third thought - I will not use black nor flake8 in prod server, that is why holding things like pytest, black, flake in development section is always a good idea.